stack.yaml.lock

```diff
@@ -0,0 +1,12 @@
+# This file was autogenerated by Stack.
```

Comment:
Let's add this file to `.gitignore` and remove it. It's not that useful here, so let's have less configuration :slightly_smiling_face: